go.mod

```diff
@@ -160,11 +166,13 @@ require (
 	go.uber.org/zap v1.17.0 // indirect
 	golang.org/x/text v0.3.7 // indirect
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	google.golang.org/genproto v0.0.0-20220126215142-9970aeb2e350 // indirect
-	google.golang.org/grpc v1.45.0 // indirect
-	google.golang.org/protobuf v1.27.1 // indirect
+	google.golang.org/genproto v0.0.0-20220426171045-31bebdecfb46 // indirect
+	google.golang.org/grpc v1.46.0 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
 )
 
 // Temporary fork for avoiding importing patent-protected code: https://github.com/hashicorp/golang-lru/issues/73
 replace github.com/hashicorp/golang-lru => github.com/ktock/golang-lru v0.5.5-0.20211029085301-ec551be6f75c
+
+replace github.com/containerd/containerd => github.com/containerd/containerd v1.6.1-0.20220428184543-bb8b134a1797
```

Comment:
Why do we need replace() ? Can we just specify this revision in require() ?

Reply:
I tried it but go get fails :(
```
 go get github.com/containerd/containerd@main
go: github.com/containerd/containerd@main (v1.6.1-0.20220504173539-885468815ef7) requires github.com/containerd/containerd@v1.6.1, not github.com/containerd/containerd@main (v1.6.1-0.20220504173539-885468815ef7)
```